src/Component/RuleTable/RuleTable.tsx

```diff
@@ -77,6 +77,12 @@ interface RuleTableState {
   filterEditorVisible: boolean;
   filterEditorPosition: DOMRect;
   hasError: boolean;
+  data: GsData;
+  rules: GsRule[];
+  countsAndDuplicates: {
```

Comment:
I'd prefer to have this split in just `counts` and `dupliactes`.

Reply:
Done in a49449f